doc/man/pam.conf-syntax.xml

```diff
@@ -297,8 +298,10 @@
         <term>done</term>
         <listitem>
            <para>
-             equivalent to ok with the side effect of terminating the module
-             stack and PAM immediately returning to the application.
+             equivalent to <emphasis>ok</emphasis> with the side effect of
+             terminating the module stack and PAM immediately returning to the
+             application in case there was no non-ignored module failure
+             before.
```

Comment:
I suggest a slightly different wording here:
```unless there was a non-ignored module failure before```
instead of
```in case there was no non-ignored module failure before```

Reply:
Fixed